rectools/models/popular.py

```diff
@@ -41,17 +43,83 @@ class Popularity(Enum):
     SUM_WEIGHT = "sum_weight"
 
 
+def _serialize_timedelta(td: tp.Optional[tp.Union[None, dict, timedelta]]) -> tp.Optional[timedelta]:
```

Comment:
It seems you've got the function names mixed up

Reply:
Fixed.